src/helpers/cliUtils.js

```diff
@@ -139,12 +148,18 @@ async function app(Client, fromDate, toDate, pathToConfig, pathToRunLogs, debug,
     const effectiveFromDate = getEffectiveFromDate(fromDate, runLogger);
 
     logger.info(`Extracting data for ${patientIds.length} patients`);
-    await extractDataForPatients(auth, config, patientIds, icareClient, messagingClient, runLogger, effectiveFromDate, toDate);
+    errors = await extractDataForPatients(auth, config, patientIds, icareClient, messagingClient, runLogger, effectiveFromDate, toDate);
   } catch (e) {
     logger.error(e.message);
     logger.debug(e.stack);
     process.exit(1);
   }
+
+  // log errors for each patient
+  Object.keys(errors).forEach((index) => {
+    logger.debug(`Errors for patient at row ${parseInt(index, 10) + 1} in .csv file:`);
```

Comment:
Could you explain this `parseInt` operation here? Why do we need the `10` there as an argument?

Reply:
The keys here are all strings so I needed to convert it to an int before adding 1 to it. I added the `10` as an argument since lint complained if I just had it as `parseInt(index)`.

Reply:
Oh, interesting; I guess that must be for the base? Okay, sounds good!

Reply:
Yea we're extending airbnb's style guide so it must be this [rule](https://github.com/airbnb/javascript#coercion--numbers)